nex/node_linux_amd64.go

```diff
@@ -12,11 +22,45 @@ func setConditionalCommands() {
 	node_preflight = nodes.Command("preflight", "Checks system for node requirements and installs missing")
 
 	node_up.Flag("config", "configuration file for the node").Default("./config.json").StringVar(&NodeOpts.ConfigFilepath)
+	node_up.Flag("metrics", "enable open telemetry metrics endpoint").Default("false").UnNegatableBoolVar(&NodeOpts.OtelMetrics)
+	node_up.Flag("otel_metrics_exporter", "OTel exporter for metrics").Default("stdout").EnumVar(&NodeOpts.OtelMetricsExporter, "stdout", "prometheus")
 	node_preflight.Flag("force", "installs missing dependencies without prompt").Default("false").BoolVar(&NodeOpts.ForceDepInstall)
 	node_preflight.Flag("config", "configuration file for the node").Default("./config.json").StringVar(&NodeOpts.ConfigFilepath)
 }
 
 func RunNodeUp(ctx context.Context, logger *slog.Logger) error {
+	if NodeOpts.OtelMetrics {
```

Comment:
This code will live nicely inside the node initialization code once we are able to merge the `feat/node` branch. I can move it in there after we get this version of it merged 👍🏻

Reply:
Perfect